src/hotspot/share/runtime/mutex.hpp

```diff
@@ -192,7 +200,7 @@ class Mutex : public CHeapObj<mtSynchronizer> {
   void print_on_error(outputStream* st) const;
   #ifndef PRODUCT
     void print_on(outputStream* st) const;
-    void print() const                      { print_on(::tty); }
+    void print() const                      { /*print_on(::tty); */ } // FIXME
```

Comment:
Can you move this print implementation into the .cpp file?

Reply:
Fixed.